tests/load/load_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// e2e implements the e2e tests.
+package precompile
```

Comment:
should this be `load`? (also updating the comment?)

Reply:
Removing this comment here as well.

Reply:
I meant, should we rename the package to `load` instead of `precompile`?